wolfcrypt/src/evp.c

```diff
@@ -2086,6 +2103,26 @@ int wolfSSL_EVP_PKEY_keygen(WOLFSSL_EVP_PKEY_CTX *ctx,
                 }
             }
             break;
+#endif
+#if !defined(NO_DH) && (!defined(HAVE_FIPS) || (defined(HAVE_FIPS_VERSION) && \
```

Comment:
Consider using the new-ish `FIPS_VERSION_GT` macro (settings.h) here and elsewhere.